xtask/src/main.rs

```diff
@@ -189,39 +299,59 @@ struct Label {
     name: String,
 }
 
+#[derive(Debug, serde::Deserialize)]
```

Comment:
Can we move this after the `PrInfo` struct so it reads top to bottom? :)

Reply:
You would have a lot of headache trying to read C code where something needs to be declared before first use. I usually prefer keeping struct and impl blocks next to each other. This shouldn't matter in the age of "Jump to definition"